ciao-controller/command.go

```diff
@@ -33,21 +33,26 @@ func (c *controller) evacuateNode(nodeID string) error {
 
 func (c *controller) restartInstance(instanceID string) error {
 	// should I bother to see if instanceID is valid?
-	// get node id.  If there is no node id we can't send a restart
 	i, err := c.ds.GetInstance(instanceID)
 	if err != nil {
 		return err
 	}
 
-	if i.NodeID == "" {
-		return types.ErrInstanceNotAssigned
-	}
-
 	if i.State != "exited" {
 		return errors.New("You may only restart paused instances")
```

Comment:
This message was not introduced by this PR.  Paused is used elsewhere, for example in the node summaries, but I was planning to remove this in a later PR.